api/proto/config/edgeview.proto

```diff
@@ -30,3 +34,9 @@ message AppDebugAccessPolicy {
     // app side of edge-view access is allowed or not
     bool allow_app = 1;
 }
+
+// Ext debug policy applicable to edge-view
+message ExtDebugAccessPolicy {
+    // external of device side of edge-view access is allowed or not
```

Comment:
I understand what dev and app policies mean. What does "ext" mean? The comments are not clear.
Is this to access downstream aka southbound devices out e.g., eth1 on the EVE-OS edge-node?

Reply:
'ext' to mean the entity external to the device, e.g. a local-profile server on the LAN outside of mgmt or app-shared ports

Reply:
Can you add that text as comments in the API.
I don't know if it would be more clear to call it "ExternalDevice" or "ExternalIP". Thoughts?

Reply:
may be 'ExternalEndPoints'